modules/adfBidAdapter.js

```diff
@@ -232,9 +232,6 @@ export const spec = {
       method: 'POST',
       url: 'https://' + adxDomain + '/adx/openrtb',
       data: JSON.stringify(request),
-      options: {
-        contentType: 'application/json'
```

Comment:
why not 'text/plain' ? This is not a change request, I'm hoping to learn about this.

Reply:
'text/plain' is the default value added by prebid core. https://github.com/prebid/Prebid.js/blob/c84cfce617b7b5128d4f34fc53087854c5b41c4b/src/adapters/bidderFactory.js#L380-L384